test/plugins/test/autolink.js

```diff
@@ -107,18 +107,17 @@ describe('AutoLink', function() {
 		assert.equal(data, '<p><span>@test</span> </p>');
 	});
 
-	it('should not create a link if last word is not a valid URL', function() {
+	it('should to try to create a link from the last word', function() {
 		testLink.call(this, {
-			expected:
-				'<p>invalid link <a href="http://ww.liferay.com">ww.liferay.com</a></p>',
-			html: 'invalid link ww.liferay.com { }',
+			expected: '<p>invalid link http:ww.liferay.com</p>',
+			html: 'invalid link http:ww.liferay.com { }',
```

Comment:
This one actually is invalid, so I think we need to keep it in a separate `it` block (ie. the old one that was there before):

```js
it('should not create a link if last word is not a valid URL', function() {
```